docs/ubuntu_install.rst

```diff
@@ -1,30 +1,31 @@
 Ubuntu Install Guide
 ====================
 
-The *Application Server* and the *Monitor Server* specifically require
-the 64-bit version of `Ubuntu Server 14.04.2 LTS (Trusty
-Tahr) <http://old-releases.ubuntu.com/releases/14.04.2/>`__. The image
-you want to get is named ``ubuntu-14.04.2-server-amd64.iso``. In order
-to verify the installation media, you should also download the files
-named ``SHA256SUMS`` and ``SHA256SUMS.gpg``.
+The *Admin Workstation*, running Tails, should be used to download and verify
+Ubuntu Server.  The *Application Server* and the *Monitor Server* specifically
```

Comment:
I agree that this is a good recommendation, since it ensures they will have the tools available (`gpg`, `sha256sum`) to verify the downloaded .iso, and it also simplifies the instructions (since we now only have to support one platform). However, it would be good to add a `.. note::` that this might take a while, since it's a large file being downloaded over Tor.